### ISSUE TITLE
No error when indenting with tabs on chained method calls

### ISSUE THREAD

**Issue by lpinca** [eslint/eslint#6901]
**What version of ESLint are you using?**

3.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

```json
{
  "rules": {
    "indent": [ "error", 2 ]
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
foo
	.bar()
  .baz();
```

**What did you expect to happen?**

An error on line 2 as it is indented with a tab not spaces.

**What actually happened? Please include the actual, raw output from ESLint.**

No error is raised.

**Comment by platinumazure** [comment]
I'm not 100% sure, but I think the `indent` rule intentionally ignores method chaining right now. @gyandeeps can you confirm?

**Comment by lpinca** [comment]
Yes I think it's ignored by default unless [`MemberExpression`](http://eslint.org/docs/rules/indent#memberexpression) is used.
The problem is that I don't want to enforce an indentation level for multi-line property chains but if there is an indentation I want it to use spaces.

**Comment by gyandeeps** [comment]
if you dont have the option on, we do not check for indention for multi-line property chain.
You can use `no-tabs` rule if you want to check for tabs.

**Comment by lpinca** [comment]
Fair enough, thanks, I'm closing.
